internal/cli/atlas/backup/restores/start.go

```diff
@@ -90,9 +90,9 @@ func (opts *StartOpts) newCloudProviderSnapshotRestoreJob() *admin.DiskBackupSna
 	if opts.oplogTS != 0 && opts.oplogInc != 0 {
 		request.OplogTs = &opts.oplogTS
 		request.OplogInc = &opts.oplogInc
-	} else if opts.pointInTimeUTCMillis != 0 {
+	} else if opts.pointInTimeUTCSeconds != 0 {
 		// Set only when oplogTS and oplogInc are not set
-		request.PointInTimeUTCSeconds = &opts.pointInTimeUTCMillis
+		request.PointInTimeUTCSeconds = &opts.pointInTimeUTCSeconds
```

Comment:
I think we should keep ` else if opts.pointInTimeUTCMillis != 0 ` as well, otherwise is not deprecating but making it not work anymore